pds_github_util/issues/move_issues.py

```diff
@@ -0,0 +1,108 @@
+import os
+import sys
+import argparse
+import logging
+import github3
+from datetime import datetime
+
+
+# Quiet github3 logging
+logger = logging.getLogger('github3')
+logger.setLevel(level=logging.WARNING)
```

Comment:
I tend not to think modules should be in control of other modules' logging because it overrides end-user desires. For example, if I were invoking a program and had set up a logging properties config file to explicitly show `github3` logs, I would be confused when I got no `github3` messages because `move_issues.py` overrides my decision.

@MJJoyce and I removed a lot of this in other parts of `pds-github-util` was [in the sandbox](https://github.com/nasa-pds-engineering-node/pds-github-util/) so I'm not sure we should re-introduce it with this PR.

Reply:
@nutjob4life @MJJoyce no problem, I think I've got that form the script I used as a template. I guess it helps to reduce the logs in the stdout but I am not sure. 

move_issue has a main function so should that be in the main function ?

Reply:
I removed the 2 lines since I don't care much about the logs.

Reply:
@nutjob4life just curious, how would you recommend we then quiet these log messages? github3 INFO logs basically tell you every line of code being executed... :-)